### ISSUE TITLE
New feature: Add option to disable auto-rotation

### ISSUE THREAD

**Issue by potwornaanakonda** [oliver-moran/jimp#214]
In some projects it would be very useful to disable auto-rotation based on EXIF metadata. E.g. when EXIF metadata is incorrect.

**Comment by strandedcity** [comment]
Agreed, it would be useful to pass an ```options``` object to ```parseBitmap(data, mime, cb)```. ```enableExifRotate: true``` (as the default when options aren't specified) would be a great first option to have.

Any interest in coding it up?

**Comment by hipstersmoothie** [comment]
proposed api:

```js
const image1 = Jimp.create({ exifRotate: true }) // off by default
const iamge2 = await Jimp.read(filePath, { exifRotate: true })
```

**Comment by porfirialex** [comment]
Is there any work around to disable this?
